src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-color-picker {
+  width: 250px;
+  border-radius: 6px;
+  box-shadow: $iui-elevation-2;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 10px;
+  padding-top: 10px;
+  @include themed {  background: rgb(255, 255, 255);}
+
+
+  .iui-color-option {
+    height: 20px;
+    width: 20px;
+    margin-right: 10px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    border-radius: 4px;
+
+    &.iui-light {
+      box-shadow: $iui-elevation-2;
+    }
+
+    &.iui-active {
+      @include themed {
+        box-shadow: 0 0 0 2pt rgb(100, 149, 237);
+      }
+    }
+
+    &:hover { 
+      @include themed {
+        box-shadow: 0 0 0 2pt rgb(170, 170, 170);
+      }
+    }
+
+    &:focus,
+    &:focus-visible {
+      @include themed {
+        box-shadow: 0 0 2pt 2pt rgb(170, 170, 170);
+      }
+    }
```

Comment:
Looking at the bigger picture, you might just be able to use the `iui-focus` mixin here.

Reply:
So if I use iui-focus, would this be instead of all the focus/focus-visible stuff I had? 

This is what using iui-focus looks like:
![image](https://user-images.githubusercontent.com/88331924/130136164-1617c133-e370-4cb5-a6f8-eec8a3e0de74.png)